kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/AsyncFetcher.java

```diff
@@ -163,6 +167,21 @@ public Builder<K, V, E> pollTimeout(long timeoutMillis) {
             return this;
         }
 
+        /**
+         * Set the {@code offsetCommitType}, options are:
+         * {@link OffsetCommitType#AUTO} let the Kafka consumer commit offsets automatically in background
+         * {@link OffsetCommitType#COMMIT_SYNC} let the Kafka consumer commit offsets synchronously after processing
+         * {@link OffsetCommitType#COMMIT_ASYNC} let the Kafka consumer commit offsets asynchronously after processing
+         * Defaults to {@code OffsetCommitType#AUTO}
+         *
+         * @param offsetCommitType {@link OffsetCommitType} enum to specify the offset commit type
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public AsyncFetcher.Builder<K, V, E> offsetCommitType(OffsetCommitType offsetCommitType) {
+            this.offsetCommitType = offsetCommitType;
```

Comment:
Nit: Please verify that the given `offsetCommitType` is not `null`. You can use the `assertNonNull` method from the `Builder#executorService` method, for that, for example.